src/components/Books.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Form from './Form';
+import Book from './Book';
+import NavBar from './NavBar';
+
+const Books = () => (
+  <>
+    <NavBar />
+    <div className="books-con">
+      <Book />
+      <Book />
+      <Book />
```

Comment:
- [ ] Good Job Creating the books component, as part of the requirements the `Book` components should take the book data as as a prop and renders the title and the author , currently the data is hard-coded inside the component , kindly fix that by passing the data through props 🙏